.gitignore

```diff
@@ -12,3 +12,4 @@ target/
 Cargo.lock
 lib*.a
 /docs/_build
+*.iml
```

Comment:
is it related to this change?

Reply:
No, this is a file generated by intellij.

Having this ignored is useful to me and possibly to others. If you agree, shall I leave it here or open a separate PR? If you don't agree, I can also remove it.